Source/ORTS.Settings/InputSettings.cs

```diff
@@ -463,6 +463,7 @@ static void InitializeCommands(UserCommandInput[] Commands)
             Commands[(int)UserCommand.DisplayCompassWindow] = new UserCommandKeyInput(0x0B);
             Commands[(int)UserCommand.DisplayHelpWindow] = new UserCommandModifiableKeyInput(0x3B, Commands[(int)UserCommand.DisplayNextWindowTab]);
             Commands[(int)UserCommand.DisplayHUD] = new UserCommandModifiableKeyInput(0x3F, Commands[(int)UserCommand.DisplayNextWindowTab]);
+            Commands[(int)UserCommand.DisplayTrainDrivingWindow] = new UserCommandModifiableKeyInput(0x3F, KeyModifiers.Control);
```

Comment:
Not keen on Ctrl+F5 as the shortcut. For our existing bindings, the general rule of thumb is that Ctrl represents some kind of mode toggle (Ctrl+F4 track monitor mode, Ctrl+F7 label display mode) while Alt opens extra windows (Alt+F9 train list window). How about Alt+F8 for the default key?

Reply:
F5 is used for the main HUD, so it seems mnemonically useful to use Ctrl-F5 for the Train Driving Info, that displays more or less the same information.

Reply:
The Train Driving Info is a small HUD, with just the information required for the driver.

Reply:
Okay, so I get the F5 mnemonic now. But it's unfortunate that this would be the only window that toggles with the Ctrl mod key. So far, for window popups, we've always used Alt:

![image](https://user-images.githubusercontent.com/4701008/125154198-b99f9580-e10d-11eb-840b-4a022171b2c9.png)

On a second look, I would actually suggest assigning the new window to Alt+F5, while moving "Basic HUD Toggle" to Shift+F5.

I should say that, in my thought process, I'm thinking about brand-new players who are downloading OR for the very first time. "Because it's the way we've always done things" isn't a convincing argument to me because, if you already have a NewYear or mainline config with Ctrl+F5 already assigned, OR will preserve, not overwrite, that assignment. Meanwhile, any new players will be left wondering about this seeming inconsistency.

Definitely a controversial decision; maybe we need the management team's opinion here? @cjakeman

Reply:
After many years using a familiar key sequence for the HUD and extended windows. I have the feeling that it won't be welcome by current users.

Reply:
> Definitely a controversial decision; maybe we need the management team's opinion here? @cjakeman

Sorry to hold you up. This mention had passed me by. Will confer and get back to you all shortly.